packages/medusa/src/api/routes/admin/auth/delete-session.ts

```diff
@@ -42,6 +42,11 @@
  *    $ref: "#/components/responses/500_error"
  */
 export default async (req, res) => {
-  req.session.destroy()
-  res.status(200).end()
+  if (req.session.customer_id) { // if we are also logged in as a customer, persist that session
```

Comment:
todo: shouldn't it be user_id here? also the comment seems a bit off

Reply:
No. The logic is correct and so is the comment. The endpoint is for admin route, so we need to only delete the admin session. So the logic is:

- If the user is logged in as both the customer and as user, delete user session (This is implemented in the if block, where the if statement checks if we are logged in as customer in addition to the user)
- else just destroy the whole session as the user is only signed in as user

Reply:
got it thanks 👍